### PR TITLE
Fix up to work with TF v1.0 API.

### PR DESCRIPTION
**Pull request by darrengarvey**
rnn_cell has moved to tensorflow.contrib.rnn. This change was made in
v0.12 [1].

[1] See https://github.com/tensorflow/tensorflow/commit/37fbebdd6c3c8f274896cc36e6feb5b7e2097a59

### CODE REVIEW

File: magenta/models/rl_tuner/rl_tuner_ops.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.contrib.rnn import rnn_cell
```
Comment (cghawthorne) [inline magenta/models/rl_tuner/rl_tuner_ops.py]: to be consistent with the rest of our code, we'd prefer to just have one `import tensorflow as tf`, and then have the below references be things like `tf.contrib.rnn.LSTMCell`.

Comment (darrengarvey) [inline magenta/models/rl_tuner/rl_tuner_ops.py]: I should have noticed that was your preferred style. I've made the change: now, do you prefer force-pushing the fixup to keep the PR clean or add an additional commit on top?

Comment (darrengarvey) [inline magenta/models/rl_tuner/rl_tuner_ops.py]: I've pushed as an extra commit, but can squash down if that's how you do it.

File: magenta/models/rl_tuner/rl_tuner_ops.py [hunk 2]
```diff
@@ -297,15 +297,17 @@ def make_rnn_cell(rnn_layer_sizes, state_is_tuple=False):
...
 
   Returns:
-      A tf.nn.rnn_cell.MultiRNNCell based on the given hyperparameters.
+      A tf.contrib.rnn.rnn_cell.MultiRNNCell based on the given
+      hyperparameters.
   """
   cells = []
   for num_units in rnn_layer_sizes:
-    cell = tf.nn.rnn_cell.LSTMCell(
+    cell = tf.contrib.rnn.rnn_cell.LSTMCell(
```
Comment (cghawthorne) [inline magenta/models/rl_tuner/rl_tuner_ops.py]: Need to remove the extra rnn_cell so it's just tf.contrib.rnn.LSTMCell

File: magenta/models/rl_tuner/rl_tuner_ops.py (hunk 1 above)
Comment (cghawthorne) [inline magenta/models/rl_tuner/rl_tuner_ops.py]: Extra commits are fine. We'll squash before we merge.

**Comment by darrengarvey** [comment]
Removed the superfluous references to `rnn_cell`. I hadn't noticed that ugliness became unnecessary.

I also fixed up a doc reference to `RnnCell`, which AFAICT should have been `RNNCell`.

**Pull request merged**